tutorials/examples/train_ising.py

```diff
@@ -81,10 +78,10 @@ def ising_n_to_ij(L, n):
     optimizer = torch.optim.Adam(gflownet.parameters(), lr=1e-3)
 
     # Learning
-    visited_terminating_states = env.States.from_batch_shape((0,))
+    env.States.from_batch_shape((0,))
```

Comment:
Does this do anything? It should be assigned.

Did you test all of the scripts (see `examples/test_scripts.py`)?

Reply:
`visited_terminating_states` is not used. Actually, the line itself is useless, so I should remove it instead(?)
Tests are working fine in both cases

Reply:
Actually this line was removed in https://github.com/GFNOrg/torchgfn/pull/203

Reply:
OK makes sense to remove it then! Thanks